pkg/relayer/header_utils.go

```diff
@@ -0,0 +1,26 @@
+package relayer
+
+import (
+	"fmt"
+	"github.com/pokt-network/poktroll/x/service/types"
+	"net/http"
+)
+
+// ForwardIdentityHeaders adds Pocket-specific identity headers from the relay metadata
+// to the HTTP request header. This helps with tracking and authentication at the
+// service backend level.
+func ForwardIdentityHeaders(header *http.Header, meta types.RelayRequestMetadata) {
```

Comment:
Does this need to be publically exposed?

if not, lowercase.

Reply:
It need to be used on the bridge for the websocket connection.
So because is part of the outside package but it need to be used on both.
If that is not enough reason let me know to figure out a way to place it in another package but anyway will need to be a public method.